<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1529</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 246, 244);</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="full_menu_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>350</width>
      <height>800</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>350</width>
      <height>800</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>15</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 163, 134);
border-radius: 10px;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="dr_dp">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>90</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>90</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resource.qrc">:/icons/icons/logo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="dr_info" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="dr_name">
             <property name="font">
              <font>
               <family>Montserrat ExtraBold</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Clincian Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="dr_desc">
             <property name="font">
              <font>
               <family>Montserrat Medium</family>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgb(255, 255, 255)</string>
             </property>
             <property name="text">
              <string>Clinician Description</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="sidebar" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
	background-color: rgb(255, 163, 134);
	border-top-right-radius: 10px;
	border-top-left-radius: 10px;
}

QPushButton{
	color: white;
	text-align: left;
	padding-left: 30px; 
	border-bottom-right-radius: 10px;
	border-top-right-radius: 10px;
	margin-top: 10px;
	border: none; 
}

QPushButton:checked {
	background-color: white;
	color: rgb(255, 163, 134);
	border-top-left-radius: 0px;
	font-weight: bold;
	border: none;
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="view_btn">
          <property name="minimumSize">
           <size>
            <width>340</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>340</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Montserrat SemiBold</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>    Patient Records</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icons/icons/w_view.png</normaloff>
            <normalon>:/icons/icons/p_view.png</normalon>:/icons/icons/w_view.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="autoRepeatDelay">
           <number>0</number>
          </property>
          <property name="autoRepeatInterval">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_btn">
          <property name="minimumSize">
           <size>
            <width>340</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>340</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Montserrat SemiBold</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>    Add Patient</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icons/icons/w_rec.png</normaloff>
            <normalon>:/icons/icons/p_talk.png</normalon>:/icons/icons/w_rec.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>36</width>
            <height>30</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="apt_btn">
          <property name="minimumSize">
           <size>
            <width>340</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>340</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Montserrat SemiBold</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>     Appointments</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icons/icons/w_cal.png</normaloff>
            <normalon>:/icons/icons/p_cal.png</normalon>:/icons/icons/w_cal.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>245</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="settings_btn">
          <property name="minimumSize">
           <size>
            <width>340</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>340</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Montserrat SemiBold</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>     Settings</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icons/icons/w_gear.png</normaloff>
            <normalon>:/icons/icons/p_gear.png</normalon>:/icons/icons/w_gear.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exit_btn">
          <property name="minimumSize">
           <size>
            <width>340</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>340</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Montserrat SemiBold</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>     Logout and Exit</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icons/icons/w_logout.png</normaloff>
            <normalon>:/icons/icons/p_logout.png</normalon>:/icons/icons/w_logout.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>1100</width>
      <height>800</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1100</width>
      <height>800</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1100</width>
      <height>800</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="view_pg_2">
        <widget class="QWidget" name="widget_2" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>1041</width>
           <height>681</height>
          </rect>
         </property>
         <widget class="QLabel" name="view_txt">
          <property name="geometry">
           <rect>
            <x>680</x>
            <y>90</y>
            <width>301</width>
            <height>601</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>25</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(124, 201, 199);</string>
          </property>
          <property name="text">
           <string>View</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QWidget" name="search" native="true">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>931</width>
            <height>51</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>348</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="search_input">
             <property name="font">
              <font>
               <family>Montserrat ExtraLight</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(228, 235, 227);
border: none; 
border-radius: 5px;
padding-left: 5px;</string>
             </property>
             <property name="text">
              <string>Enter Patient Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="search_btn">
             <property name="font">
              <font>
               <family>Montserrat Light</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	border:none; 
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icons/icons/p_mag.png</normaloff>:/icons/icons/p_mag.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>348</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QScrollArea" name="patient_scroll_area">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>180</y>
            <width>561</width>
            <height>311</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 163, 134);
font: 57 9pt &quot;Montserrat Medium&quot;;
color: white;</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>559</width>
             <height>309</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="gender_label">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="name_label">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="add_pg">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="add">
           <property name="font">
            <font>
             <pointsize>25</pointsize>
            </font>
           </property>
           <property name="text">
            <string>add</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="apts_pg">
        <widget class="QLabel" name="apts_txt">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>1106</width>
           <height>701</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="text">
          <string>apts</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="settings_pg">
        <widget class="QLabel" name="settings_txt">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>1106</width>
           <height>701</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="text">
          <string>settings</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>search_btn</tabstop>
  <tabstop>search_input</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
